HWLesson5/Task2.js

```diff
@@ -0,0 +1,27 @@
+function User(obj) {
```

Comment:
Пропоную використати тут деструктеризацію
```suggestion
function User({ name, age, followers }) {
```